instrumentation/netty-reactor-0.8.0/src/main/java/com/nr/instrumentation/reactor/netty/TokenLinkingSubscriber.java

```diff
@@ -67,8 +67,10 @@ private void withNRToken(Runnable runnable) {
     @Trace(async = true, excludeFromTransactionTrace = true)
     private void withNRError(Runnable runnable, Throwable throwable) {
         if (token != null && token.isActive()) {
-            token.linkAndExpire();
+            token.link();
             if (NettyReactorConfig.errorsEnabled) {
+                // I believe 100% of users disable this as it makes NewRelic to report caught and handled
+                // exceptions as errors in APM. Is there a real use case for this?
```

Comment:
Would you be OK with setting the default to false and leaving the config in case someone finds it useful

Reply:
I just wanted to draw your attention to this. No change is necessary if you think it may be useful for someone.

Please also note that there is a similar lifter in spring-webflux instrumentation module. It reports such error unconditionally and expires tokens immediately after linking (one of the issues being fixed in this PR).
https://github.com/newrelic/newrelic-java-agent/blob/2ff0a81c4903854ea64f313422c9e520fa0cdf94/instrumentation/spring-webflux-5.3.0/src/main/java/com/nr/agent/instrumentation/spring/reactive/Util.java#L74-L92
I use spring webflux server and spring webclient with reactor-netty within the same application. Given that neither token expiration nor error reporting happens after I apply this PR and configure reactor-netty instrumentation to not report errors, it seems that the webflux's lifter is not used and creates a lot of confusion.

Reply:
Thanks for the pointer, I'll revisit that instrumentation and see if we can't clean it up some